docs/Project-Configuration/Project-and-Distro-Settings.md

```diff
@@ -84,12 +84,27 @@ Admins can also set the branch project to inherit values from a
 repo-level project settings configuration. This can be learned about at
 ['Using Repo Level Settings'](Repo-Level-Settings).
 
+#### Spawn Host Script Path
+
+The spawn host script path provides an option to run a script from your repo
+when a spawn host starts. This can be useful for workflows where users typically
+want . This should be specified as a file path relative to the project root.
```

Comment:
I think this sentence is incomplete (want what?

Reply:
I'm not sure where I was going with this sentence, so I removed. The description of how it behaves is good enough.